client/implementation/pom.xml

```diff
@@ -67,8 +67,8 @@
         </dependency>
 
         <dependency>
-            <groupId>jakarta.json</groupId>
```

Comment:
This dependency was repeated. There are a few other dependencies where I'm not sure about their scope (some should be provided)